cluster-provision/k8s/1.24/k8s_provision.sh

```diff
@@ -55,9 +55,11 @@ cat <<EOF >/etc/NetworkManager/conf.d/002-dhclient.conf
 dhcp=dhclient
 EOF
 
-sysctl -w net.netfilter.nf_conntrack_max=1000000
+
 echo "net.netfilter.nf_conntrack_max=1000000" >> /etc/sysctl.conf
 
+sysctl --system
```

Comment:
maybe better to move this glued to line 59
since they belong

Reply:
Tidied up this and the empty line.